src/pages/docs/projects/steps/conditions/index.mdx

```diff
@@ -108,16 +108,10 @@ When you review a process with two steps that run in parallel, you'll notice two
 
 ### Maximum parallelism
 
-To help your Octopus Server remain stable, we have defaulted the maximum number of steps you can run concurrently in any given deployment to `10`. You can configure any number of steps to run in parallel, and Octopus will roll through them as quickly as possible, but it will only run `10` of them at any one time by default.
+There is no limitation on the number of steps that can run in parallel. 
 
-#### Overriding maximum parallelism
+The `Octopus.Action.MaxParallelism` variable allows for specifying the number of machines on which the action will concurrently execute which is the same variable used for [Rolling Deployments](https://octopus.com/docs/deployments/patterns/rolling-deployments).
```

Comment:
Is it worth making it clear that the default parallelism of 10 still applies for the number of targets a given step will execute on?

Eg.:


```
When a step executes on multiple deployment targets, it will execute on up to 10 targets in parallel by default.
You can change this number by setting the `Octopus.Action.MaxParallelism` variable. This variable also controls [Rolling Deployments](https://octopus.com/docs/deployments/patterns/rolling-deployments)
```

Reply:
this is called out other places in the docs but happy to mention it here too but was trying to avoid including a number there for the skim readers

https://octopus.com/docs/projects/variables/system-variables#user-modifiable-settings

Reply:
Cool. It was just a drive-by opinion.